### ISSUE TITLE
Implement TPM_CC_PolicyAuthorizeNV

### ISSUE THREAD

**Issue by flihp** [tpm2-software/tpm2-tss#682]
This command was added in v1.38 of the TPM spec. See list of command codes in table 12 here: https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf

**Comment by williamcroberts** [comment]
When are you pulling this over to master? I need it.

**Comment by williamcroberts** [comment]
FYI on [line 150](https://github.com/tstruk/tpm2-tss/commit/758fd13180606f851dedab5822989c7f9761cdcc#diff-f9748c23d9fe53fac6e1d32a81094794L150) the table should be 12 right ([spec](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf))?

**Comment by tstruk** [comment]
@williamcroberts later this week. I can create a separate PR with the command today if that unblocks you, but still working on tests. We have a separate issue #735 about the table numbers.

**Comment by williamcroberts** [comment]
@tstruk I really just need the `#define` for the CC code.

**Comment by tstruk** [comment]
@williamcroberts Here is the PR: https://github.com/tpm2-software/tpm2-tss/pull/776
